api/models/topics.models.js

```diff
@@ -5,3 +5,15 @@ exports.selectTopics = () => {
         SELECT * FROM topics
         ;`);
 };
+
+exports.selectArticleById = (req) => {
+	id = req.params.article_id;
```

Comment:
Anything to do with req and res should be dealt with in the controller. Could you please move this there? 👍